### PR TITLE
email pdf statement to patients with email address

### PR DESCRIPTION
**Pull request by venambati**
Hello Brady,

Here is the code that emails the patient statement to patients who allow emails and has a valid email address. It also shows that if the user has an email and allows email in the "EOB Posting - Search" page.

Please let me know if there are any changes that needs to be done.

Thank you
Venkat Ambati

### CODE REVIEW

**Comment by venambati** [comment]
@sjpadgett ah, that was a cut and paste mistake, I should not have removed $_POST['form_portalnotify'] I will fix that and repush it.

**Comment by sjpadgett** [comment]
@venambati Thank you

File: interface/billing/sl_eob_search.php [hunk 1]
```diff
@@ -148,6 +148,47 @@ function bucks($amount) {
...
+        return false;
+    }
+    if (!(validEmail($GLOBALS['patient_reminder_sender_email']))) {
+        return false;
+    }
+
+    $mail = new MyMailer();
+    $pt_name=$patientData['fname'].' '.$patientData['lname'];
+    $pt_email=$patientData['email'];
+    $email_subject=xl('Peak Cardiology Patient Statement Bill');
```
Comment (bradymiller) [inline interface/billing/sl_eob_search.php]: Above appears to be a custom clinic name

File: interface/billing/sl_eob_search.php [hunk 2]
```diff
@@ -873,6 +937,17 @@ function npopup(pid) {
      	}?>
    </td>
    <?php } ?>
+  <td class="detail" align="left">
+   <?php
+    $patientData = sqlQuery("SELECT * FROM `patient_data` WHERE `pid`=?", array($row['pid']) );
+    if ( $patientData['hipaa_allowemail'] == "YES" && $patientData['allow_patient_portal'] == "YES" && $patientData['hipaa_notice'] == "YES" && validEmail($patientData['email'])) {
+        echo "YES";
```
Comment (bradymiller) [inline interface/billing/sl_eob_search.php]: Does this need to be translated? (ie. is the user seeing this?)

File: interface/billing/sl_eob_search.php [hunk 3]
```diff
@@ -873,6 +937,17 @@ function npopup(pid) {
...
+    } else {
+        echo "NO";
```
Comment (bradymiller) [inline interface/billing/sl_eob_search.php]: Does this need to be translated? (ie. is the user seeing this?)

**Comment by bradymiller** [comment]
Hi @venambati ,
This will be a very nice feature. Just let me know when next revision is ready for review (it should be ready for the codebase at that point).
thanks! -brady

**Comment by bradymiller** [comment]
Hi @venambati ,

I just brought this into the codebase. thanks for the contribution!

Note I did make a couple minor changes(just translation stuff) here:
https://github.com/openemr/openemr/commit/fbc7377b9d4d7e25728a504a83ffa304c9eb11e1

I think something that would likely be helpful is logging when email was sent(in case need to track if emails were sent).

-brady

**Pull request merged**